app/javascript/controllers/chapter_controller.js

```diff
@@ -237,14 +266,14 @@ export default class extends Controller {
   changeFont(font) {
     let path = this.activeTab.find(".pagination").data("url");
     let verseList = this.activeTab;
-    const isReadingMode = false;
-
-    fetch(`${path}?${$.param({ font: font, reading: isReadingMode })}`)
+    const reading = window.history.state?.reading || false;
```

Comment:
@mahmoudfelfel 

- Click on reading mode 
- Change the fonts
  - Reading mode will be updated with selected font 
- Now click on translations mode again 
  - You'll see old fonts,  its because this tab was rendered using old font, and we didn't update it. 

I'll suggest updating the active tab, and replace the inactive tab with the following lazy load DOM.

```
<div class="render-async  " id="RANDOM-ID" data-path="/1/1-7?reading=TRUE|FALSE" data-method="GET" data-headers="{}" data-lazy-load='{"root":"#pill-reading|#pill-translations"}' data-controller="render-async">
            <p class="text-center p-3">
              <i class="fa fa-spinner fa-spin"></i> Loading...
</p>
</div> 
```